src/quo/components/inputs/amount_input/view.cljs

```diff
@@ -0,0 +1,86 @@
+(ns quo.components.inputs.amount-input.view
```

Comment:
while it's true this is an input, we follow the same hierarchy as in Figma to make the component library easier to main.

<img width="287" alt="Screenshot 2024-01-31 at 22 11 06" src="https://github.com/status-im/status-mobile/assets/22799766/d0fd41bd-c55a-447a-8523-571788427c6b">

the name space here should be 
`quo.components.wallet.amount-input.view`
same for style file and component-spec

Reply:
Great! Done ✅